### DISCUSSION TITLE
[PLUGIN] Swap "enter" and "shift enter"

### DISCUSSION THREAD

**Discussion by Nriver** [TriliumNext/Trilium#6042] [Show and tell]
Hi there! I’ve created a widget that swaps the functions of "Enter" and "Shift + Enter." Feel free to check it out here: [swap-enter-widget](https://github.com/Nriver/swap-enter-widget).

**Comment by maphew** [comment]
Is this is to fix the "problem" of too much whitespace being added between lines of text? If so: yay!

I've wondered about how to change that myself. My idea was to change the CSS to narrow the default whitespace. Did you consider this too, and decide against it? if so, what made changing hard and soft line breaks better?

**Reply by Nriver** [reply]
I did not think that much. It's just some user ask me to do this swap and I made it happen. I don't have strong opinions on the spacing.
